coalib/output/ConsoleInteraction.py

```diff
@@ -833,8 +847,29 @@ def ask_for_action_and_apply(console_printer,
                                  failed_actions,
                                  result,
                                  file_diff_dict,
-                                 file_dict)
+                                 file_dict,
+                                 applied_actions)
         return ret
+    if action_name == 'ShowAppliedPatchesAction':
+        show_patch_action = ShowPatchAction()
+
+        for key, val in applied_actions.items():
+            this_result = val[0]
```

Comment:
I hate that 0 and 3 are hardcoded. Can u make some constants somewhere like result_index, in case we reuse them? Its easier to maintain